### DISCUSSION TITLE
Hi! CAN29bit protocol cant recieve messages.

### DISCUSSION THREAD

**Discussion by IlyaSolomka** [BrianHumlicek/J2534-Sharp#35] [Q&A]
This is channel options `protocol = Protocol.CAN;
txFlag = TxFlag.CAN_29BIT_ID;
connectionFlag = ConnectFlag.CAN_29BIT_ID;
baud = Baud.CAN;`
Sending is ok, but no recieved messages, always "BUFFER_EMPTY" status. 
Can some body help me?

**Comment by 33ev** [comment]
try another j2534 device

**Comment by IlyaSolomka** [comment]
but another sofware can recieve 29bit messages whith exactly this device...

**Comment by BrianHumlicek** [comment]
29bit CAN is picky about 29bit flags being set.  Ensure on sent messages, and filters you have the 29bit flag set.

Brian
